config/article.js

```diff
@@ -5,7 +5,7 @@
  * @TODO Please ensure this file is filled out before publishing!!!
  */
 
-export default (environment = 'development') => {
+module.exports = (environment = 'development') => {
```

Comment:
If this works for LOAS that's fine, I'm not going to suggest you revert all these because it's like the only Webpack-based project we still have going, but would renaming all these files to .mjs instead of converting to CJS be a better approach (if we're ultimately trying to avoid shipping CJS whereever possible)?